src/main/kotlin/model/GameResultStatus.kt

```diff
@@ -0,0 +1,20 @@
+package model
+
+class GameResultStatus(player: AbstractPlayer, list: List<AbstractPlayer>) {
+    var playerName: PlayerName = player.name
```

Comment:
프로퍼티의 getter 함수를 노출하면서 setter 함수에 대한 접근을 제한하려면 setter 함수를 private으로 지정하면 됩니다.

```suggestion
    var playerName: PlayerName = player.name
        private set
```

Reply:
반영하였습니다.